update header project list accessibility

flora8984461 opened the pull request:
According to https://github.com/EclipseFdn/solstice-assets/issues/221#issuecomment-761183342

Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

chrisguindon commented on less/_components/discover-search.less:
```diff
@@ -36,3 +36,22 @@
 }
 
+.featured-jumbotron {
+  .form-search-projects {
```
I would prefer that we implement a solution that would allow us to re-use these styles for other sites.

I am thinking that should replace .form-search-projects with .input-group

Also, I don't think that all of these css properties are needed if we remove the .textfield-underline  class from the .form-control.

flora8984461 replied:
.textfield-underline class is also used here (following image), Do you also wanna change this one? 

![image](https://user-images.githubusercontent.com/39588094/105076000-82063980-5a58-11eb-9826-b57f961d2aa0.png)

flora8984461 replied:
I noticed that on page: https://accounts.eclipse.org/contact/membership we are also using input-group. Do we want to change them all to border-rounded?

flora8984461 commented on less/_components/classes.less:
```diff
@@ -83,28 +83,19 @@ img.desaturate {
-  &:-moz-placeholder { /* Firefox 18- */
-    color: inherit;
+// Example: project page search
+.input-group {
```
I'm moving it inside here, since it's a general input class, not just for project discover

chrisguindon replied:
The problem now is that this will affect all .input-group on the site and we only want to modify the jumbotron...

flora8984461 commented on less/_components/quicksilver/featured-jumbotron.less:
```diff
@@ -124,4 +124,8 @@
       color:@brand-primary;
     }
   }
+
+  #project-list-link {
+    color: @white;
+  }
 }
```
Move it to jumbotron

flora8984461 commented on docs/index.html:
```diff
@@ -745,7 +760,7 @@ <h2>Sign up to our Newsletter</h2>
           <form action="https://www.eclipse.org/donate/process.php" method="post" target="_blank">
             <div class="form-group">
               <input type="hidden" name="type" value="newsletter">
-              <input type="email" value="" name="email" class="textfield-underline form-control" id="mce-EMAIL" placeholder="Email">
+              <input type="email" value="" name="email" class="form-control" id="mce-EMAIL" placeholder="Email">
```
No idea do we need this one? I don't see where it's used in our sites. Probably I'm missing Drupal sites?

flora8984461 replied:
Just a note here, if I do not do anything else, just removed that class here, it becomes:

![image](https://user-images.githubusercontent.com/39588094/105080865-6eaa9c80-5a5f-11eb-9d85-37730de4805e.png)


But if this is not used anywhere, it's fine, we can remove it.

chrisguindon reviewed (changes requested):
Let's keep .textfield-underline in the css, however, I want us to stop using it in the jumbotron html.

chrisguindon reviewed (changes requested):
I think we need to restrict these css changes to the jumbotron..

flora8984461 commented:
> I think we need to restrict these css changes to the jumbotron..

Sorry, just updated. I think I misunderstood 😅

chrisguindon commented on less/_components/quicksilver/featured-jumbotron.less:
```diff
@@ -124,4 +124,24 @@
+  }
+
+  #project-list-link {
+    color: @white;
```
I don't think we should include an #id here. I am thinking the solution here is to simply set the default color of https://github.com/EclipseFdn/solstice-assets/blob/fc39ee526a072eedce05dc406b35491dc216b81a/less/_components/quicksilver/featured-jumbotron.less#L57 to white...

flora8984461 replied:
But will that affect the other sites? 🤔

flora8984461 replied:
Is there a way that I can add overwrite class and variables in /projects/ page? Like what we do for the Hugo sites? 🤔

chrisguindon replied:
Yes, but I think it will make them better... All sites can override that value on a per-site basis. Our background image was lighter before and the previous color did work well but I think we might need to change it now since our background image are darker...

flora8984461 replied:
So to confirm, you wanna change the default to white that will take effect on all the other sites?

chrisguindon reviewed (changes requested):
see previous comment

flora8984461 commented:
> see previous comment

Just updated the default link color for jumbotron.
under .featured-jumbotron a:not(.btn):not(.jumbotron-icon) and changed @jumbotron-link-color-visited and @jumbotron-link-color default color to white.
